### PR TITLE
Email Address Top Nav

### PR DESCRIPTION
**Pull request by jessedoyle**
:elephant::art:

* Accept a `display_name` attribute for a navigation instance that will be displayed in the top navbar. If the name is nil, them the email address of the user is displayed. This content is truncated to 30 characters.

### CODE REVIEW

**Comment by jessedoyle** [comment]
:wave:

File: lib/ama_layout/decorators/navigation_decorator.rb [hunk 1]
```diff
@@ -6,6 +6,10 @@ def items
       object.items.map { |i| i.decorate }
     end
 
+    def display_name_text
+      (display_email? && email || "Welcome, #{display_name}").truncate(30)
```
Comment (MathieuGilbert) [inline lib/ama_layout/decorators/navigation_decorator.rb]: This looks like the perfect place for a ternary.

File: lib/ama_layout/navigation.rb [hunk 2]
```diff
@@ -33,6 +34,18 @@ def navigation_items
       YAML.load ERB.new(File.read nav_file_path).result
     end
 
+    def display_name
+      @display_name && @display_name.titleize || email
```
Comment (MathieuGilbert) [inline lib/ama_layout/navigation.rb]: Personally I'd rather see a ternary than the logical equivalent.

`display_email? ? email : @display_name.titleize`

**Comment by MathieuGilbert** [comment]
💩

**Comment by jessedoyle** [comment]
:wave:

**Comment by rubene** [comment]
🐘 🐘 🐘

**Comment by rubene** [comment]
🎨 🎨 🎨

**Pull request merged**